src/main/java/org/sagebionetworks/bridge/exporter/request/BridgeExporterRequest.java

```diff
@@ -324,26 +360,23 @@ public BridgeExporterRequest build() {
                 throw new IllegalStateException("startDateTime must be before endDateTime.");
             }
 
-            // If start/endDateTime are specified, there must be a studyWhitelist.
-            if (hasStartDateTime && studyWhitelist == null) {
-                throw new IllegalStateException("If start- and endDateTime are specified, studyWhitelist must also " +
-                        "be specified.");
```

Comment:
As written, if you have a start/endDateTime, we currently still require studyWhitelist.